substrafl_examples/get_started/run_mnist_torch.py

```diff
@@ -44,24 +44,32 @@
 
 N_CLIENTS = 3
 
-# Every computation will run in `subprocess` mode, where everything runs locally in Python
-# subprocesses.
-# Other backend_types are:
-# "docker" mode where computations run locally in docker containers
-# "remote" mode where computations run remotely (you need to have a deployed platform for that)
-# To run in remote mode you have to also use the function `Client.login(username, password)`
 client_0 = Client(backend_type="subprocess")
 client_1 = Client(backend_type="subprocess")
 client_2 = Client(backend_type="subprocess")
 
+# %%
+# Every computation will run in `subprocess` mode, where everything runs locally in Python
+# subprocesses.
+# Other backend_types are:
+#
+# - `docker` mode where computations run locally in docker containers
+# - `remote` mode where computations run remotely (you need to have a deployed platform for that)
+#
+# To run in remote mode, use the following syntax:
+#
+# `client_remote = Client(url="MY_BACKEND_URL")`
+#
+# `client_remote.login(username="my-username", password="my-password")`
```

Comment:
I can't manage to have this displayed properly. It's either:

Reply:
![image](https://user-images.githubusercontent.com/5354301/219308467-c91c642f-1ee0-4b48-bf0b-bfa63f26be45.png)

Reply:
Or 
```
# `client_remote = Client(url="MY_BACKEND_URL")`
#
# `client_remote.login(username="my-username", password="my-password")`
```
![image](https://user-images.githubusercontent.com/5354301/219308535-b2375fbc-a65f-4bf7-a879-bb8d59280807.png)

Reply:
@ThibaultFy @SdgJlbl any idea?

Reply:
And with double backticks? I _think_ it's the way to display code in RsT 🤔